src/handbook/sales/meetings/poc.md

```diff
@@ -0,0 +1,20 @@
+## POC
+
+Every POC call should have an engineer present.
```

Comment:
I think the Cloud trial is fine without, maybe move this to `Self-hosted trial` and say the installation should have an engineer present?